### PR TITLE
Issue#14: Make snapshot of chain data

### LINKED ISSUE #14: Add function that make a snapshot
Backup chain data
- Command: `archway chain snapshot`
- Store location: `/archway-cli/snapshot/2022-05-19T07:59:58.676Z_data.zip`

### PR DESCRIPTION
**Pull request by Choi-Jinhong**
Add command about making snapshot of chain data.

close #14

### CODE REVIEW

File: src/commands/chain.js [hunk 1]
```diff
@@ -82,6 +82,19 @@ async function main(archwayd, name, options = {}) {
...
             });	
             break;
+        case 'snapshot': 
+            const today = new Date();
+            console.log("Make a snapshot");
+            const mkDir = spawnSync("mkdir", [`${archwayd.archwaydHome}/snapshot`]);
+            if (mkDir.error != null){
+                throw mkDir.error;
+            }
+            const snapshot = spawnSync("tar", ["zcvf", `${archwayd.archwaydHome}/snapshot/${today.getFullYear()}${today.getMonth()}${today.getDate()}_data.tar.gz`,`${archwayd.archwaydHome}/data`]);
```
Comment (rootwarp) [inline src/commands/chain.js:92]: As I remember, we discuss that store new snapshot into `archway` project directory.
But as I see here created snapshot stores into chain's home.

Would you keep this impl. or change that?
I think there's no problem whatever if you're ok.

**Comment by rootwarp** [comment]
@Choi-Jinhong Sorry. 
Regarding to [this](https://github.com/dsrvlabs/archway-cli/pull/15#pullrequestreview-978132184)`Users/rootwarp/.archway` should be removed.
You should assume the user will unzip under the home dir.

File: src/commands/chain.js [hunk 2]
```diff
@@ -82,6 +85,19 @@ async function main(archwayd, name, options = {}) {
...
             break;
+        case 'snapshot':
+            const rootPath = path.dirname(await Config.path(this.pathPrefix));
+            const nowDate = new Date();
+            console.log("Make a snapshot");
+            const mkDir = spawn("mkdir", [`${rootPath}/snapshots`]);
+            if (mkDir.error != null){
+                throw mkDir.error;
+            }
+            const snapshot = spawn("tar", ["zcvf", `${rootPath}/snapshots/chaindata_${nowDate.toISOString()}.tar.gz`,`${archwayd.archwaydHome}/data`, "-C", "~/.archway"]);
```
Comment (rootwarp) [inline src/commands/chain.js:96]: "~/.archway" hardcoded here.
It should be retrieved from `archwayd` because home directory can be changed.

Comment (rootwarp) [reply src/commands/chain.js]: Try to use `tar -C ~/.archway -zcvf test.tar.gz data`
This tested on OSX and Linux.

**Review by hqueue — APPROVED** [review]
@Choi-Jinhong LGTM Thanks!

**Review by rootwarp — APPROVED** [review]
LGTM :)

**Pull request merged**